azure/Guidelines.md

```diff
@@ -87,11 +87,11 @@ http(s)://<tenant-id>-<service-defined-root>.<service>.azure.net
 
 All Azure APIs **MUST** use explicit versioning.  It's critical that clients can count on services to be stable over time, and it's critical that Azure services can add features and make changes.
 
-The OneAPI guidelines offer a couple of different options on how to specify an API version and guidance on what constitutes a breaking change.  This section of the Azure API guidelines describes which options are required of Azure services as well as some guidance about deprecation policy.  There is also a section about additional versioning practices necessary to support Azure Stack and Azure compatibility.
+The Microsoft API guidelines offer a couple of different options on how to specify an API version and guidance on what constitutes a breaking change.  This section of the Azure API guidelines describes which options are required of Azure services as well as some guidance about deprecation policy.  There is also a section about additional versioning practices necessary to support Azure Stack and Azure compatibility.
```

Comment:
The docs is called the Microsoft REST API guidelines - we have lots of other API guidelines.

Reply:
Grammar nits: Consider removing "a couple of" and "as some". They're just extra words that don't add anything.

Reply:
I didn't like the way this was worded - so I word-smithed.  I think it displays intent better now.